lib/BackbeatConsumer.js

```diff
@@ -511,10 +514,38 @@ class BackbeatConsumer extends EventEmitter {
     }
 
     /**
-    * force commit the current offset and close the client connection
-    * @param {callback} cb - callback to invoke
-    * @return {undefined}
-    */
+     * pause the kafka consumer
+     * @param {string} site - name of site
+     * @return {undefined}
+     */
+    pauseService(site) {
```

Comment:
In this class there's no notion of service, just a consumer. So the function names can be simply "pause" and "resume".